### PR TITLE
Refactoring: introduce message view controller 

### PR DESCRIPTION
**Pull request by vadymmarkov**
`MessageViewController` is used instead of `InfoView` that we had before. Here I use constraints instead of frame calculation, so we get back nice animation between `loading` and `scanning` states.

### CODE REVIEW

File: Sources/Controllers/MessageViewController.swift [hunk 1]
```diff
@@ -1,5 +1,212 @@
...
+    handleStateUpdate()
+  }
+
+  public override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+    blurView.frame = view.bounds
+  }
+
+  public override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
```
Comment (onmyway133) [inline Sources/Controllers/MessageViewController.swift]: 🤔

File: Sources/Controllers/MessageViewController.swift [hunk 2]
```diff
@@ -1,5 +1,212 @@
...
+    borderView.layer.borderWidth = 2
+    borderView.layer.cornerRadius = 10
+    borderView.layer.borderColor = UIColor.black.cgColor
+  }
+
+  private func handleStateUpdate() {
+    borderView.isHidden = true
+    borderView.layer.removeAllAnimations()
+
+    switch state {
```
Comment (onmyway133) [inline Sources/Controllers/MessageViewController.swift]: 👍

File: Sources/Extensions/UIView+Extensions.swift [hunk 3]
```diff
@@ -8,4 +8,14 @@ extension UIView {
     
     return .zero
   }
+
+  func addSubviews(_ subviews: UIView...) {
```
Comment (onmyway133) [inline Sources/Extensions/UIView+Extensions.swift]: Sugar sugar 😬

**Comment by onmyway133** [comment]
Nice PR

File: Sources/Controllers/MessageViewController.swift (hunk 1 above)
Comment (vadymmarkov) [reply Sources/Controllers/MessageViewController.swift]: Will remove this in the next PR.

**Pull request merged**